02-peer-protocol.md

```diff
@@ -787,9 +787,21 @@ is destined, is described in [BOLT #4](04-onion-routing.md).
 #### Requirements
 
 A sending node:
-  - MUST NOT offer `amount_msat` it cannot pay for in the
-remote commitment transaction at the current `feerate_per_kw` (see "Updating
-Fees") while maintaining its channel reserve.
+  - if it is _responsible_ for paying the Bitcoin fee:
+    - MUST NOT offer `amount_msat` if, after adding that HTLC to its commitment
+    transaction, it cannot pay the fee for either the local or remote commitment
+    transaction at the current `feerate_per_kw` while maintaining its channel
+    reserve (see [Updating Fees](#updating-fees-update_fee)).
+    - SHOULD NOT offer `amount_msat` if, after adding that HTLC to its commitment
+    transaction, its remaining balance doesn't allow it to pay the fee for a
```

Comment:
I also realized this can be interpreted as the remaining balance must be enough for (non-dust HTLC value + fee for this HTLC). Maybe we should just explicitly state that it should be enough to pay the fee to receive another HTLC.

Reply:
True, I'll try re-wording that a bit.

Reply:
Done in https://github.com/lightningnetwork/lightning-rfc/pull/740/commits/afb7313e0f81bfee32aaa9b71fbf6c512407e8d1